pkg/exchange/ftx/exchange.go

```diff
@@ -0,0 +1,107 @@
+package ftx
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+const (
+	restEndpoint       = "https://ftx.com"
+	defaultHTTPTimeout = 15 * time.Second
+)
+
+type Exchange struct {
+	rest *restRequest
+}
+
+func NewExchange(key, secret string, subAccount string) *Exchange {
```

Comment:
is subaccount optional?

Reply:
Yes. If your credential is created for all accounts, you don't have to specify that. If it's only for subaccount, you just specify it.